packages/synapse-react-client/src/components/QueryWrapperPlotNav/RowSetView.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { Suspense, useState } from 'react'
+import { CardContainer } from '../CardContainer/CardContainer'
+import { SynapseTable, SynapseTableConfiguration } from '../SynapseTable'
+import { CardConfiguration } from '../CardContainerLogic'
+import LastUpdatedOn from './LastUpdatedOn'
+import { useDeepCompareMemoize } from 'use-deep-compare-effect'
+import QueryWrapperLoadingScreen from '../QueryWrapper/QueryWrapperLoadingScreen'
+import { SynapseErrorBoundary } from '../error'
+import { useQueryContext } from '../QueryContext'
+import { Box } from '@mui/material'
+import { TablePagination } from '../SynapseTable/TablePagination'
+import { ViewMoreQueryResultsButton } from '../QueryWrapper/ViewMoreQueryResultsButton'
+import { useRowSet } from './UseRowSet'
+
+export type RowSetViewProps = {
+  tableConfiguration?: SynapseTableConfiguration
+  cardConfiguration?: CardConfiguration
+  initialLimit?: number
+  hideDownload?: boolean
+}
+
+/**
+ * Renders a table or cards representing the RowSet. Also displays appropriate pagination controls
+ * depending on the value of QueryContext.isInfinite.
+ * @param props
+ * @constructor
+ */
+export function RowSetView(props: RowSetViewProps) {
```

Comment:
Handles rendering a `RowSet` (as a table or cards) and pagination/infinite data for that `RowSet`. Replaces `FilterAndView` (which did not do any filtering, so it needed a better name).